packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -3,108 +3,58 @@
 // found in the LICENSE file.
 
 import 'package:meta/meta.dart';
+import 'package:process/process.dart';
 
 import '../base/common.dart';
+import '../base/config.dart';
 import '../base/file_system.dart';
+import '../base/io.dart';
+import '../base/logger.dart';
 import '../base/os.dart';
 import '../base/platform.dart';
 import '../base/process.dart';
 import '../base/version.dart';
 import '../convert.dart';
-import '../globals.dart' as globals;
 import 'android_studio.dart';
 
 // ANDROID_HOME is deprecated.
 // See https://developer.android.com/studio/command-line/variables.html#envar
 const String kAndroidHome = 'ANDROID_HOME';
 const String kAndroidSdkRoot = 'ANDROID_SDK_ROOT';
 
-// Android SDK layout:
-
-// $ANDROID_SDK_ROOT/platform-tools/adb
-
-// $ANDROID_SDK_ROOT/build-tools/19.1.0/aapt, dx, zipalign
-// $ANDROID_SDK_ROOT/build-tools/22.0.1/aapt
-// $ANDROID_SDK_ROOT/build-tools/23.0.2/aapt
-// $ANDROID_SDK_ROOT/build-tools/24.0.0-preview/aapt
-// $ANDROID_SDK_ROOT/build-tools/25.0.2/apksigner
-
-// $ANDROID_SDK_ROOT/platforms/android-22/android.jar
-// $ANDROID_SDK_ROOT/platforms/android-23/android.jar
-// $ANDROID_SDK_ROOT/platforms/android-N/android.jar
 
 final RegExp _numberedAndroidPlatformRe = RegExp(r'^android-([0-9]+)$');
 final RegExp _sdkVersionRe = RegExp(r'^ro.build.version.sdk=([0-9]+)$');
 
 /// The minimum Android SDK version we support.
 const int minimumAndroidSdkVersion = 25;
 
-/// Locate ADB. Prefer to use one from an Android SDK, if we can locate that.
-/// This should be used over accessing androidSdk.adbPath directly because it
-/// will work for those users who have Android Platform Tools installed but
-/// not the full SDK.
-String getAdbPath([ AndroidSdk existingSdk ]) {
```

Comment:
I removed these wrappers and inlined the logic into the adb/et cetera getters on the android SDK. Isn't necessary if we don't inject null